mitiq/executor/tests/test_executor.py

```diff
@@ -182,16 +183,14 @@ def test_run_executor_force_run_all_serial_executor_identical_circuits(
 @pytest.mark.parametrize("s", (50, 100, 150))
 @pytest.mark.parametrize("b", (1, 2, 100))
 def test_run_executor_preserves_order(s, b):
-    rng = np.random.RandomState(1)
-
     collector = Executor(executor=executor_batched_unique, max_batch_size=b)
     assert collector.can_batch
 
     circuits = [
         cirq.Circuit(cirq.H(cirq.LineQubit(0))),
         cirq.Circuit([cirq.H(cirq.LineQubit(0))] * 2),
     ]
-    batch = [circuits[i] for i in rng.random_integers(low=0, high=1, size=s)]
+    batch = choices(circuits, k=s)
```

Comment:
@andreamari is it fine to remove the seed here? I don't think it's needed, but want your opinion.

Reply:
Fine to remove the seed. Any random ordering is ok.